docs/user-guide/catalog.md

```diff
@@ -218,6 +225,11 @@ Enabling that option, all layers added to the map from this catalog source will
 
 * *Layer tile size*: it represents tile size (width and height) to be used for tiles of all layers added to the map from the catalog source (`256x256` or `512x512`). For layers already loaded on the map, it is possible to change the tile size through the [Layer Settings](https://mapstore.readthedocs.io/en/latest/user-guide/layer-settings/#display) tool as usual.
 
+* *Server Type*: to choose the type of server from which the url of the entered CSW service comes. The options are two: `Geoserver` or `No Vendor`which can be mapproxy, mapserver or other.
```

Comment:
CSW not involved here @ElenaGallo. The purpose of this is to be able to specify if the WMS server behind is a GeoServer or not.